modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -771,12 +772,65 @@ the following property.
 headers. Valid options: `true` or `false`
 | `ignoreEmptyLine` | Boolean | `true` a| Indicates whether to ignore empty
 line. Valid options: `true` or `false`
+| `streaming` | Boolean | `false` a| Introduced in Mule 4.2.2: Streaming is intended for
+processing a large file. When streaming is enabled, the reader accesses each row
+sequentially, keeping one row in memory at a time, instead of making all data
+available at once. Streaming does not permit random access to rows in the file.
 | `tableOffset` | String | None a| The position of the first cell in the
 table (`<Column><Row> example A1`).
 | `zipBombCheck` | Boolean | `true` a| If set to `false`, the zip bomb check is
 turned off. Valid options: `true` or `false`
 |===
 
+[[stream_csv]]
+.Streaming Example:
+By default, the Excel reader stores input data from an entire file in-memory
+if the file is 1.5MB or less. If the file is larger than 1.5 MB, the process
+writes the data to disk. For very large files, you can improve the performance
+of the reader by setting a `streaming` property to `true`. To demonstrate the use of
+this property, the next example streams a XLSX file and transforms it to JSON.
+
+The following example streams an Excel file and transforms it to JSON.
```

Comment:
These two sentences are similar:
To demonstrate the use of this property, the next example streams a XLSX file and transforms it to JSON.
The following example streams an Excel file and transforms it to JSON

I would live just the first one.